composer.json

```diff
@@ -15,13 +15,18 @@
 		"php": ">=5.6"
 	},
 	"require-dev": {
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.6",
 		"roave/security-advisories": "dev-master",
-		"wptrt/wpthemereview": "^0.2"
+		"wptrt/wpthemereview": "*",
+		"dealerdirect/phpcodesniffer-composer-installer": "0.6",
```

Comment:
Please use the caret to get the latest patch version of the tool. Versions 0.6.0 and 0.6.1 have known bugs. 0.6.2 is the latest and will be used automatically when using the caret for the versioning.

```suggestion
		"dealerdirect/phpcodesniffer-composer-installer": "^0.6",
```

Reply:
Thank you! The caret got lost due to copy and paste.